### ISSUE TITLE
Ionide removes properties from .fsproj

### ISSUE THREAD

**Issue by Inkp** [ionide/ionide-vscode-fsharp#679]
### Problem
Ionide seems to remove project propeties from .fsproj file

### Steps to reproduce

1. Install VS code
2. Install ionide
3. Create new netcore F# app, e.g. `dotnet new console -o test -lang F#`
3. Open folder with .fsproj file in VS code
4. Manually insert attribute into .fsproj file e.g. `<ServerGarbageCollection>true</ServerGarbageCollection>`
5. Add new file via F# project explorer
6. Watch `<ServerGarbageCollection>` attribute dissapear from .fsproj

**Comment by enricosada** [comment]
I can repro.

It replace it also if the file was saved (not just dirty).

**Comment by Krzysztof-Cieslak** [comment]
Forge tries to model the project files really strictly and probably just doesn't know about `ServerGarbageCollection` property. Should be an easy fix.

**Comment by isaacabraham** [comment]
I also saw this with Zebra with it removing some attribute (a different one) that tries to automatically read files based on globs rather than explicit list.

**Comment by weebs** [comment]
Same issue occurs with values in `<PropertyGroup>`. 

Original fsproj:

```xml
<Project Sdk="Microsoft.NET.Sdk">
  <PropertyGroup>
    <RootNamespace>fs_playground</RootNamespace>
    <OutputType>Exe</OutputType>
    <TargetFramework>netcoreapp2.1</TargetFramework>
    <FscToolPath>C:\Program Files (x86)\Microsoft Visual Studio\2017\Professional\Common7\IDE\CommonExtensions\Microsoft\FSharp</FscToolPath>
    <FscToolExe>fsc.exe</FscToolExe>
  </PropertyGroup>
  <ItemGroup>
    <Compile Include="Types.fs" />
    <Compile Include="Program.fs" />
  </ItemGroup>
  <Import Project=".paket\Paket.Restore.targets" />
</Project>
```

After moving Types.fs with the down arrow in the UI

```xml
<Project Sdk="Microsoft.NET.Sdk">
  <PropertyGroup>
    <RootNamespace>fs_playground</RootNamespace>
    <OutputType>Exe</OutputType>
    <TargetFramework>netcoreapp2.1</TargetFramework>
  </PropertyGroup>
  <ItemGroup>
    <Compile Include="Program.fs" />
    <Compile Include="Types.fs" />
  </ItemGroup>
  <Import Project=".paket\Paket.Restore.targets" />
</Project>
```

**Comment by ZaymonFC** [comment]
This also happens to me with the `<ProjectId>` tag.
After reordering files the tag is removed.

**Comment by pblasucci** [comment]
Add `<LangVersion>...</LangVersion>` to the list of tags which are getting stripped out.

**Comment by Krzysztof-Cieslak** [comment]
Should be fixed with 5.0
